src/evidently/analyzers/stattests/ks_stattest.py

```diff
@@ -8,7 +8,11 @@
 from evidently.analyzers.stattests.registry import StatTest, register_stattest
 
 
-def _ks_stat_test(reference_data: pd.Series, current_data: pd.Series, threshold: float) -> Tuple[float, bool]:
+def _ks_stat_test(
+        reference_data: pd.Series,
+        current_data: pd.Series,
+        _feature_type: str,
```

Comment:
can you rename `_feature_type` to `feature_type` and add a new parameter to docstrings?